### PR TITLE
Convert links to ReDown

### PR DESCRIPTION
**Pull request by TheTripleV**

### CODE REVIEW

**Comment by jasondaming** [comment]
Looks like there is some issue with a link ending in a )

**Comment by TheTripleV** [comment]
I don't know why just the read the docs build is failing on a warning:
> /home/docs/checkouts/readthedocs.org/user_builds/frc-docs/checkouts/2690/source/index.rst:453: WARNING: toctree contains reference to nonexisting document 'docs/beta/beta-getting-started/trackers-reporting-bugs'

I don't see that warning locally

**Comment by sciencewhiz** [comment]
I believe that comes from changing a file thats ignored (which the beta docs are right now)

**Comment by TheTripleV** [comment]
How would I fix it? Should I revert changes to the whole beta folder?

**Comment by TheTripleV** [comment]
the linkcheck failure isn't real

File: source/docs/contributing/frc-docs/style-guide.rst [hunk 1]
```diff
@@ -207,7 +207,7 @@ It is preferred to format external links as anonymous hyperlinks. The important
 
 .. code-block:: ReST
```
Comment (sciencewhiz) [inline source/docs/contributing/frc-docs/style-guide.rst:208]: Should this be md now?

File: source/docs/contributing/frc-docs/style-guide.rst [hunk 2]
```diff
@@ -3,7 +3,7 @@
 Style Guide
 ===========
 
-This document contains the various RST/Sphinx specific guidelines for the frc-docs project. For guidelines related to the various WPILib code projects, see `the WPILib GitHub <https://github.com/wpilibsuite/styleguide>`__
+This document contains the various RST/Sphinx specific guidelines for the frc-docs project. For guidelines related to the various WPILib code projects, see [the WPILib GitHub](https://github.com/wpilibsuite/styleguide)
```
Comment (sciencewhiz) [inline source/docs/contributing/frc-docs/style-guide.rst:6]: This should reference that we use RST and md

File: source/docs/contributing/frc-docs/style-guide.rst [hunk 3]
```diff
@@ -3,7 +3,7 @@
...
 
 Filenames
```
Comment (sciencewhiz) [inline source/docs/contributing/frc-docs/style-guide.rst:8]: Should new files keep using .rst, or change?

**Review by sciencewhiz — COMMENTED** [review]
Did you use a script to make these conversions? If so it might be good to check in. There's several decent size PRs that are open that will need their links converted.

**Comment by TheTripleV** [comment]
> Did you use a script to make these conversions? If so it might be good to check in. There's several decent size PRs that are open that will need their links converted.

I did a vscode global find/replace with 
> find: (?<!`)`(?!`)(.*?) <(.*?)>(?<!`)`(?!`)__
replace: `[$1]($2)`

It's success rate is probably 95% correct, 1% bad, 4% missed

File: source/docs/contributing/frc-docs/style-guide.rst (hunk 3 above)
Comment (TheTripleV) [reply source/docs/contributing/frc-docs/style-guide.rst:8]: I'm open to either (keep files as .rst or change to .rd).
Changing the file extension will break all open PRs so I'm more weary of it

**Comment by sciencewhiz** [comment]
It looks like that search didn't work for links without titles

```
  C:\Users\chris\Documents\Robotics\frc-docs\source\docs\software\basic-programming\coordinate-system.rst (2 hits)
	Line 340: .. [#] Rapid React field image from MikLast on Chiefdelphi `<https://www.chiefdelphi.com/t/2022-top-down-field-renders/399031>`__
	Line 341: .. [#] CHARGED UP field image from MikLast on Chiefdelphi `<https://www.chiefdelphi.com/t/2023-top-down-field-renders/421365>`__
  C:\Users\chris\Documents\Robotics\frc-docs\source\docs\software\vision-processing\wpilibpi\installing-the-image-to-your-microsd-card.rst (1 hit)
	Line 10: <https://github.com/wpilibsuite/WPILibPi/releases>`__.
  C:\Users\chris\Documents\Robotics\frc-docs\source\docs\yearly-overview\known-issues.rst (1 hit)
	Line  47: If you identify software that interferes with driver station, please post it to `<https://github.com/wpilibsuite/allwpilib/issues/6174>`__
  C:\Users\chris\Documents\Robotics\frc-docs\source\docs\zero-to-robot\step-4\creating-test-drivetrain-program-cpp-java-python.rst (1 hit)
	Line 112: .. note:: the Python example below is from `<https://github.com/robotpy/examples/tree/main/GettingStarted>`__
```

**Comment by TheTripleV** [comment]
Those should stay for now as they are a special case.

We could do [](link) as the syntax for that. But most markdown flavors support raw links without any special syntax.

**Comment by sciencewhiz** [comment]
My only concern is that we now have a mixture of rst link syntax and md link syntax. If those were raw links (which sphinx appears to support too), then I think we'd be ok.

**Comment by TheTripleV** [comment]
I'm fine with a little bit of rst syntax lingering in these automation forward prs and getting cleaned up later in smaller, manual prs. Maybe those links should have titles? Idk.

**Pull request merged**